app/provider/app.go

```diff
@@ -654,6 +663,8 @@ func New(
 	// NOTE: Capability module must occur first so that it can initialize any capabilities
 	// so that other modules that want to create or claim capabilities afterwards in InitChain
 	// can do so safely.
+	// NOTE: The provider module must come after genutils and staking, since it relies on the
+	// information about the validators these modules provide to compute validator updates.
 	app.MM.SetOrderInitGenesis(
```

Comment:
Do we need to have the `no_valupdates_*` modules here as well?

Reply:
Same as above: no